plugin/src/main/scala/com/typesafe/genjavadoc/AST.scala

```diff
@@ -122,7 +122,7 @@ trait AST { this: TransformCake ⇒
         case m: MethodSymbol => m.isVarargsMethod
         case _               => false
       }
-      val d = newDefDef(sym, EmptyTree)()
+      val d = DefDef(sym, EmptyTree)
```

Comment:
Yes, it passes all the tests with all cross scala versions. Unfortunately Mr. Travis build got a timeout while switching to the latest cross scala version.